File: app/src/main/java/com/tammamkhalaf/myuaeguide/categories/hereDeveloper/discoverExplore/Location.java

package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Location {

    @SerializedName("position")
    @Expose
    private List<Double> position = null;
    @SerializedName("address")
    @Expose
    private Address address;

    public List<Double> getPosition() {
        return position;
    }

    public void setPosition(List<Double> position) {
        this.position = position;
    }

    public Address getAddress() {
        return address;
    }

    public void setAddress(Address address) {
        this.address = address;
    }

}

File: shared/src/commonMain/kotlin/com/tammamkhalaf/kmmsharedmodule/categories/hereDeveloper/discoverExplore/Location.kt
package com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore

import com.google.gson.annotations.Expose

class Location {
    @SerializedName("position")
    @Expose
    private var position: List<Double>? = null

    @SerializedName("address")
    @Expose
    private var address: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Address? = null
    fun getPosition(): List<Double>? {
        return position
    }

    fun setPosition(position: List<Double>?) {
        this.position = position
    }

    fun getAddress(): com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Address? {
        return address
    }

    fun setAddress(address: com.tammamkhalaf.myuaeguide.categories.hereDeveloper.discoverExplore.Address?) {
        this.address = address
    }
}